test/src/unit-capi-empty-var-length.cc

```diff
@@ -218,6 +248,22 @@ void StringEmptyFx::write_array(const std::string& array_name) {
       &buffer_a3_offsets_size,
       buffer_a3,
       &buffer_a3_size);
+  if (rc != TILEDB_OK) {
+    tiledb_error_t* error;
+    tiledb_ctx_get_last_error(ctx, &error);
+    const char* msg;
+    tiledb_error_message(error, &msg);
+    printf("=========== %s\n", msg);
```

Comment:
looks like a dev print, should this make it into the PR?

Reply:
removed, thanks